scripts/risk_pipeline/data_prep/coinbase_data_prep.py

```diff
@@ -0,0 +1,47 @@
+import argparse
+import os
+import get.coinbase as gc
+import treat.coinbase_treatment as tc
+
+
+def get_params():
+    """
+    Get parameters from command line
+    """
+    parser = argparse.ArgumentParser()
+
+    parser.add_argument(
```

Comment:
Copy pasting code as you are doing here and in Historical Data (all the argparse code) is a smell